hack/postgres/README.md

```diff
@@ -7,9 +7,9 @@ Scripts and helpers for working with postgres
 #### Prerequisites
 ```
 export ns=cloud-resources-load-test
-# Spin up workshop `postgres` instance
+# Spin up openshift `postgres` instance
```

Comment:
similar to previous comments -  Could we somehow talk about two other Providers?